src/app/utils/profilesPaths.ts

```diff
@@ -0,0 +1,12 @@
+import en from 'i18n/resources/en/extension.json';
+import fr from 'i18n/resources/fr/extension.json';
+export default {
+  en: {
+    CONTRIBUTORS: en.path.profiles.contributors,
```

Comment:
Is this still used? You could probably remove this file.
If you need to force a language you can still use the `lng` param:
```
t('i18n', { lng: 'en' } ); 
```